src/main/java/com/nike/cerberus/service/AuthenticationService.java

```diff
@@ -268,19 +273,50 @@ private IamRoleAuthResponse authenticate(IamPrincipalCredentials credentials, Ma
         return iamRoleAuthResponse;
     }
 
+    private AuthTokenResponse createToken(String principal,
```

Comment:
repeat about Hystrix metrics

Reply:
Todd has a ticket in the backlog or on his desk for Hystrix and RDS. Right now we don't have that integration